test/unit/manipulation.js

```diff
@@ -3099,3 +3099,55 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
+
+	// Define the expected number of assertions
+
+	assert.expect( 4 );
+
+	// Set up the HTML structure by appending to the existing fixture
+
+	var fixture = jQuery( "#qunit-fixture" );
+
+	fixture.append( `
+	<div id="container">
+	  <div class="guarded removeself" data-elt="one">
+	    Guarded 1
+	  </div>
```

Comment:
Indentation is still incorrect. For example, in these few lines, there are:
* tabs mixed with spaces - we only use tabs
* the string inside of the call is indented as the call above - it should be indented more

Also, these tests need to work in IE 11, so no ES 6 is allowed. You need to use regular concatenated strings, as other tests do.

Reply:
Done

Reply:
This has not been done; in particular, the last section:
> Also, these tests need to work in IE 11, so no ES 6 is allowed. You need to use regular concatenated strings, as other tests do.

Reply:
done